src/structs/prompt.rs

```diff
@@ -169,7 +170,7 @@ impl Prompt<'_> {
                         if cursor_pos > 0 {
                             cursor_pos -= 1;
                             term.flush()?;
-                        } else if search_term.is_empty() {
+                        } else if search_term.is_empty() && self.items[0].ends_with("..") {
```

Comment:
fix left arrow on topmost directory enter the first item instead of doing nothing